src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { UserIdContext, ActiveTodoContext } from '../../util/Store';
+import { getUser } from '../../api';
+import { Todo, User } from '../../types/Types';
 
 export const TodoModal: React.FC = () => {
+  const { activeUser, setActiveUser } = useContext(UserIdContext);
+  const { todo, setTodo } = useContext(ActiveTodoContext);
+  const [user, setUser] = useState<User>();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getUser(activeUser as number)
```

Comment:
```suggestion
    getUser(todo.userId)
```
you can just use the todo.userId here